I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 34, in script
    x "Hola, yo soy el narrador, les contare la historia sobre la batalla de Naruto y Sasuke contra Kakashi"
IOError: Couldn't find file 'images/kana.ogg'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 34, in script
    x "Hola, yo soy el narrador, les contare la historia sobre la batalla de Naruto y Sasuke contra Kakashi"
  File "C:\Users\Home\Desktop\PROGRAMACION_1\SoftwareProgra1\renpy-6.99.5-sdk\renpy\ast.py", line 594, in execute
    renpy.exports.say(who, what, interact=self.interact)
  File "C:\Users\Home\Desktop\PROGRAMACION_1\SoftwareProgra1\renpy-6.99.5-sdk\renpy\exports.py", line 1021, in say
    who(what, interact=interact)
  File "C:\Users\Home\Desktop\PROGRAMACION_1\SoftwareProgra1\renpy-6.99.5-sdk\renpy\character.py", line 826, in __call__
    self.do_display(who, what, cb_args=self.cb_args, **display_args)
  File "C:\Users\Home\Desktop\PROGRAMACION_1\SoftwareProgra1\renpy-6.99.5-sdk\renpy\character.py", line 688, in do_display
    **display_args)
  File "C:\Users\Home\Desktop\PROGRAMACION_1\SoftwareProgra1\renpy-6.99.5-sdk\renpy\character.py", line 491, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "C:\Users\Home\Desktop\PROGRAMACION_1\SoftwareProgra1\renpy-6.99.5-sdk\renpy\ui.py", line 277, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\Home\Desktop\PROGRAMACION_1\SoftwareProgra1\renpy-6.99.5-sdk\renpy\display\core.py", line 2285, in interact
    repeat, rv = self.interact_core(preloads=preloads, **kwargs)
  File "C:\Users\Home\Desktop\PROGRAMACION_1\SoftwareProgra1\renpy-6.99.5-sdk\renpy\display\core.py", line 2572, in interact_core
    renpy.audio.audio.interact()
  File "C:\Users\Home\Desktop\PROGRAMACION_1\SoftwareProgra1\renpy-6.99.5-sdk\renpy\audio\audio.py", line 790, in interact
    periodic()
  File "C:\Users\Home\Desktop\PROGRAMACION_1\SoftwareProgra1\renpy-6.99.5-sdk\renpy\audio\audio.py", line 724, in periodic
    c.periodic()
  File "C:\Users\Home\Desktop\PROGRAMACION_1\SoftwareProgra1\renpy-6.99.5-sdk\renpy\audio\audio.py", line 340, in periodic
    topf = load(self.file_prefix + topq.filename + self.file_suffix)
  File "C:\Users\Home\Desktop\PROGRAMACION_1\SoftwareProgra1\renpy-6.99.5-sdk\renpy\audio\audio.py", line 74, in load
    rv = renpy.loader.load(fn)
  File "C:\Users\Home\Desktop\PROGRAMACION_1\SoftwareProgra1\renpy-6.99.5-sdk\renpy\loader.py", line 512, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'images/kana.ogg'.

Windows-8-6.2.9200
Ren'Py 6.99.5.602
Naruto_LuisAlvarez 0.0
